src/tools/docker-to-docker-compose/docker-to-docker-compose.vue

```diff
@@ -0,0 +1,64 @@
+<template>
+  <n-card>
+    <n-input
+      v-model:value="dockerRun"
+      style="font-family: monospace"
+      type="textarea"
+      placeholder="Your docker run command to convert..."
+      rows="3"
+    />
```

Comment:
Wrap with a `<n-form-item label="Your docker run command:">` for accessibility and UX

Reply:
Done